lib/galaxy/util/__init__.py

```diff
@@ -721,7 +721,7 @@ def __init__(self, params, sanitize=True):
                     key not in self.NEVER_SANITIZE and
                     True not in [key.endswith("|%s" % nonsanitize_parameter) for
                                  nonsanitize_parameter in self.NEVER_SANITIZE]):
```

Comment:
@dannon The visual break is there only because of the more-indented list comprehension being at the end of the condition, try to swap the last 2 conditions:

```python
                if (value is not None and
                    True not in [key.endswith("|%s" % nonsanitize_parameter) for
                                 nonsanitize_parameter in self.NEVER_SANITIZE] and
                    key not in self.NEVER_SANITIZE):
                    self.__dict__[key] = sanitize_param(value)
```

I know it's valid, but if we aim at consistency then the double indentation always works:
```python
                if (value is not None and
                        key not in self.NEVER_SANITIZE and 
                        True not in [key.endswith("|%s" % nonsanitize_parameter) for
                                     nonsanitize_parameter in self.NEVER_SANITIZE]):
                    self.__dict__[key] = sanitize_param(value)
```

@jmchilton Fine for me.